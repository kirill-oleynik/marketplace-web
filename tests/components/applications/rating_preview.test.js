const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const {
  RatingPreview
} = require('../../../app/components/applications/rating_preview');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <RatingPreview
        t={(translation) => translation}
        total={1}
        average={1}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
