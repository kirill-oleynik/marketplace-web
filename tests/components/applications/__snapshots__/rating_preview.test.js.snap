// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="rating mb-30"
>
  <div
    className="rating__stars rating__stars-1 mr-15"
  >
    <i
      className="rating__stars-item icon icon-star"
    />
    <i
      className="rating__stars-item icon icon-star"
    />
    <i
      className="rating__stars-item icon icon-star"
    />
    <i
      className="rating__stars-item icon icon-star"
    />
    <i
      className="rating__stars-item icon icon-star"
    />
  </div>
  <span
    className="align-middle"
  >
    1.0
    <span
      className="in-black-050"
    >
      appProfile.ratings.totalVotes
    </span>
  </span>
</div>
`;
